fastly/resource_fastly_tls_subscription.go

```diff
@@ -213,6 +218,9 @@ func resourceFastlyTLSSubscriptionRead(_ context.Context, d *schema.ResourceData
 		domains = append(domains, domain.ID)
 	}
 
+	// NOTE: there must be only one certificate id included per subscription
+	var certificateId = subscription.Certificates[0].ID
```

Comment:
`TestAccResourceFastlyTLSSubscription` is failing on this line with this error:
```
panic: runtime error: index out of range [0] with length 0
```
Should we generate a certificate for that test case?

Reply:
Thanks @Hiieu 
No, I think I need to add some conditions there. Will test and get back to you.

Reply:
@Hiieu pushed another commit.

Reply:
Nice, looks good to me 👍